docs/timeline.md

```diff
@@ -2,4 +2,48 @@
 title: Using the Timeline View
 ---
 
-TODO:
+* toc
+{:toc}
+
+## What is it?
+The timeline view displays information about Flutter frames. It consists of
+ three parts, each increasing in granularity:
+- Frame Rendering Chart
+- Frame Flame Chart
+- Event Details
+
+**Use a profile build of your application to analyze performance.** Frame rendering times are not indicative of release
+performance unless your application is run in profile mode.
+
+## Frame Rendering Chart
+This chart is populated with individual frames as they are rendered in your application. Each bar in the chart
+represents a frame. The bars are color-coded to highlight the different portions of work that occur when rendering a
+Flutter frame:
+- CPU: Dart code running from your application as well as the the Flutter framework
+- GPU: graphics code running from the Flutter Engine
+
+<img src="images/timeline_frame_rendering_chart.png" width="800" />
+
+### Jank
+The frame rendering chart shows CPU and GPU jank in shades of red. We consider a portion of the frame to be janky when
+it takes more than 8 ms to complete.
+
+To achieve a frame rendering rate of 60 FPS (frames per second), each frame must render in ~16 ms or less. Since there
+are two portions of work for each frame (CPU and GPU), each portion should complete in 8 ms or less. When this target is
+missed, you may experience UI jank or dropped frames.
+
+## Frame Flame Chart
+The flame chart shows the event trace for a single frame. The top-most event spawns the event below it, and so on and so
+forth. The CPU and GPU events are separate event flows, but they share a common timeline (displayed at the top of the
+flame chart). This timeline is strictly for the given frame. It does not reflect the clock shared by all frames.
+
+<img src="images/timeline_flame_chart.png" width="800" />
+
+The flame chart supports zooming and scrolling. You can click an event to view more details in the details section below
```

Comment:
You way additionally want to mention how to zoom and pan.

Reply:
done.